### ISSUE TITLE
Parse error

### ISSUE THREAD

**Issue by jlecordier** [saper150/nginx-format#9]
Hi,

It's me again, with a real issue this time.

This part in my config is causing a `Parsing Error` :

```nginx
    map $imgformat $pimgformat {
        default "png";
        ".png" "png";
        ".jpeg" "jpeg";
    }
```

This part is inside the `http` block

**Comment by saper150** [comment]
should be fixed now in 1.8 version

**Comment by jlecordier** [comment]
@saper150 You're the best 🚀
